### ISSUE TITLE
Neue Beteiligtenliste

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#750]
Erst einmal herzlichen Dank für neue Version von J-Lawyer und das ganze Engagement und die Zeit die dahintersteckt. Ich bin gerade dabei die Beteiligtenlisten auszubauen. Dabei ist mir aufgefallen, dass wenn man einen neuen Beteiligten anlegt, die Liste der vorhandenen Beteiligten sich nicht sortieren lässt, z.B. durch klicken auf Namen. Das wird unter Umständen schnell unübersichtlich. Wäre eine kleine Verbesserung der super Verbesserung :-)

**Comment by j-dimension** [comment]
@Mucki526 wäre das evtl. mit #740 erledigt?

**Comment by Mucki526** [comment]
Nein, ich meinte das Fenster, in dem ich die neuen Beteiligten überhaupt erzeuge, nicht das Fenster mit den Beteiligten in der Akte. Wenn ich will die verschiedenen Beteiligten erzeugen, um sie in Dokumentvorlagen zu verwenden. Wenn ich nicht mehr weis, wieviel Kläger 1 bis Kläger x ich schon erzeugt habe, dann sortiere ich über den Namen und sehe aha, es gibt erst Kläger 1 und 2, nächster wäre Kläger 3. Im Moment ist diese Liste unsortiert nach der Reihenfolge der Anlage der neuen Beteiligten. Das ist unübersichtlich und sortiert sich auch nicht, wenn man neu einsteigt.
